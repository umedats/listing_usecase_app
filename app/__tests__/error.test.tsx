import React from "react";
import { render, screen } from "@testing-library/react";
import ErrorRender from "@/app/[locale]/error";

describe("Error Component", () => {
  it("renders an error message", () => {
    const error = new Error("Test error message");
    const reset = jest.fn();
    const { asFragment } = render(<ErrorRender error={error} reset={reset} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
