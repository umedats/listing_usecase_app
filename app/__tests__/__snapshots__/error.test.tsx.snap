// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Component renders an error message 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthMd css-xdmu94-MuiContainer-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-rl3d6r-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
      >
        <h6
          class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
        >
          Something went wrong!
        </h6>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
